packages/commonwealth/client/scripts/state/api/threads/deleteReaction.ts

```diff
@@ -37,12 +42,11 @@ const deleteReaction = async ({ reactionId, threadId }: DeleteReactionProps) =>
   }
 };
 
-const useDeleteThreadReactionMutation = () => {
+const useDeleteThreadReactionMutation = ({ chainId, threadId }: IuseDeleteThreadReactionMutation) => {
   return useMutation({
     mutationFn: deleteReaction,
     onSuccess: async (response) => {
-      // TODO: when we migrate the reactionCounts store proper to react query
-      // then we will have to update the react query state here
+      updateThreadInAllCaches(chainId, threadId, { associatedReactions: [{ id: response.data.result.reaction_id }] as any }, 'removeFromExisting')
```

Comment:
can we fix the type instead of any?

Reply:
Unfortunately the API only returns IDs and we only need those to remove from cache. The whole object is in state, so if we want to fix we would have to loop it over and full data from data and then extract ids -- so I just used `any` here instead to avoid code smells from future.